### ISSUE TITLE
ImageUploaderInline upload fails with "Please correct the error below"

### ISSUE THREAD

**Issue by monospacesoftware** [inventare/django-image-uploader-widget#147]
I followed your tutorial for using `ImageUploaderInline` in a ModelAdmin, adding `inlines = [ProductImageAdmin]`.  The widget displays, and I can upload one or more images. But when I click Save, the form shows "Please correct the error below." There is no more information about an error.  There is nothing in the Django logs. The image I uploaded now shows up as a white box.
Can you tell me where or how I can debug this? Using `Django==5.0.3`

**Comment by monospacesoftware** [comment]
Well I narrowed it down to an issue with Safari.  Error does not occur using Chrome.  Safai=16.5.2.

In the form data upload, the image field does not have a filename:

```
------WebKitFormBoundary2ppXZmXtDv9RTVyU
Content-Disposition: form-data; name="images-0-image"; filename=""
Content-Type: application/octet-stream
```

**Comment by EduardoJM** [comment]
Hello, thanks for reporting

Really, looks a problem with Safari... 

I will to make some tests to check if is a failure of the side of this application and try to found a work-around. Maybe, this can be relative to the move the image or clone temporary file input to inlines at https://github.com/inventare/django-image-uploader-widget/blob/main/image_uploader_widget/static/admin/js/image-uploader-inline.js#L215

**Comment by EduardoJM** [comment]
Hi @monospacesoftware, i released 0.4.2 at PyPI, can you upgrade to it and try if it fix your problem?

I made some tests using my iOS phone Safari and apparently it fixed the problem. Can you validate with your Safari?

**Comment by monospacesoftware** [comment]
Uploading does work in Safari on version 0.4.2, thank you. I did find some other bugs however, although they are not as critical. Can open another ticket if you prefer.

* Drag and drop reordering does not work in Safari.There's no message or error in the console. The drag and drop thing just doesn't drop the image and change the order.
* If you click Add Image, upload an Image, then click the Delete button on the image, then upload the same image again, nothing happens.  It seems to work ok if you upload a different image after the delete. There's no error in the console.

**Comment by EduardoJM** [comment]
Thanks for the new reports. Don't need to open new issue for this. I will work on these small fixes on this week.

**Comment by EduardoJM** [comment]
> * Drag and drop reordering does not work in Safari.There's no message or error in the console. The drag and drop thing just doesn't drop the image and change the order.

Fixed it on `0.4.4`, like this (i finnaly installed a MacOS on a virtual machine):

https://github.com/inventare/django-image-uploader-widget/assets/20397491/e14808cc-669c-4a8a-a28b-dca05a01a718

@monospacesoftware  Can you validate if upgrading to `0.4.4` really solve those problems?

**Comment by monospacesoftware** [comment]
I think it's all working as expected now!  good job, thanks.
